File: Development/Projeto1/Projeto1/ViewController.m
//
//  ViewController.m
//  Projeto1
//
//  Created by Jonas Junior on 06/08/21.
//

#import "ViewController.h"
#import "NextViewController.h"

@interface ViewController ()



@end

@implementation ViewController

//- (void)viewDidLoad {
//    [super viewDidLoad];
//    // Do any additional setup after loading the view.
//}
- (IBAction)nextPage:(id)sender {
    NSLog(@"Indo para a próxima página");
    
    NextViewController *nextViewController = [[NextViewController alloc] init];
    
    [self presentViewController:nextViewController
                    animated:YES
                    completion:nil];
}

-(void) AlertComDecisao: (NSString * ) msg {
  UIAlertController * alert = [UIAlertController
                                 alertControllerWithTitle: @"Tem certeza?"
                                 message: msg
                                 preferredStyle: UIAlertControllerStyleAlert
                                ];
  UIAlertAction * action = [UIAlertAction
                            actionWithTitle: @"Cancelar"
                            style: UIAlertActionStyleCancel
                            handler: ^ (UIAlertAction * _Nonnull action) {
                            NSLog(@ "Cancelando");
                            }
                           ];
    UIAlertAction * action2 = [UIAlertAction
                              actionWithTitle: @ "Confirmar"
                              style: UIAlertActionStyleDestructive
                              handler: ^ (UIAlertAction * _Nonnull action2) {
                              NSLog(@ "Confirmando");
                              }
                             ];
    [alert addAction: action];
    [alert addAction: action2];
  [self presentViewController: alert animated: true completion: nil];
}

- (IBAction)btnAlert:(id)sender {
    [self AlertComDecisao:@"Se eu fosse você cancelaria!"];
}
@end

File: Development/taskContinue/taskContinue/ViewController.swift
//
//  ViewController.swift
//  taskContinue
//
//  Created by Jonas Junior on 09/08/21.
//

import UIKit


class ViewController: UIViewController {
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
  
       
    }

    @IBAction func alertUser(_ sender: Any) {
        print("alertando")
        
        let alert = UIAlertController(
            title: "Tem certeza que deseja continuar?",
            message: "Confirmar apagará seu progresso",
            preferredStyle: .alert)
        
        alert.addAction(UIAlertAction(
                            title: NSLocalizedString("Confirmar",
                            comment: "Default action"),
                            style: .destructive,
                            handler: { _ in NSLog("O \"Confirmar\" foi pressionado.")
        }))
        
        alert.addAction(UIAlertAction(
                            title: NSLocalizedString("Cancelar",
                            comment: "Default action"),
                            style: .cancel,
                            handler: { _ in NSLog("O \"Cancelar\" foi pressionado.")
        }))
        self.present(alert, animated: true, completion: nil)
    }
    
    @IBAction func nextView(_ sender: Any) {
        print("Indo para a próxima página")
        
        let vc = storyboard?.instantiateViewController(identifier: "newVC") as! NewViewController
        present(vc, animated: true)
        
    }
}


